mixins/demo/localize-test.js

```diff
@@ -25,7 +25,6 @@ class LocalizeTest extends LocalizeMixin(LitElement) {
 			'ja': { 'hello': 'こんにちは {name}' },
 			'ko': { 'hello': '안녕하세요 {name}' },
 			'pt-br': { 'hello': 'Olá {name}' },
-			'sv': { 'hello': 'Hallå {name}' },
```

Comment:
I removed Swedish from the demo/test component so that the tests can test the scenario where there are resources but they're missing an entry for the document language that `formatNumber` wants to use.